sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcWriteSchemaTransformProvider.java

```diff
@@ -73,8 +78,29 @@ public JdbcWriteSchemaTransform(JdbcWriteSchemaTransformConfiguration config) {
     }
 
     protected JdbcIO.DataSourceConfiguration dataSourceConfiguration() {
+      String driverClassName = config.getDriverClassName();
+
+      if (Strings.isNullOrEmpty(driverClassName)) {
```

Comment:
Same as above, put these in a map. (Seems like it's the same map. Refactor to a common location?)

Reply:
Done.